src/test/java/uk/gov/hmcts/befta/dse/ccd/DefinitionConverterTest.java

```diff
@@ -57,38 +60,41 @@ void testMainIncorrectArgs() {
 	 */
 	@Test
 	void testMainIncorrectArgspos() {
-		String[] args = {"to-json","definition_files","true"};
+        String[] args = { "to-json", "definition_files", CcdEnvironment.PREVIEW.name(), "true" };
 		IllegalArgumentException aeThrown = Assertions.assertThrows(IllegalArgumentException.class, () ->
         	DefinitionConverter.main(args),
 				"IllegalArgumentException is not thrown");
-		assertTrue(aeThrown.getMessage().contains("Third arg should be a path not a boolean, if you wish to set Jurisdiction"));
+        assertTrue(aeThrown.getMessage()
+                .contains("Fourth arg should be a path not a boolean, if you wish to set Jurisdiction"));
 	}
 
 	/**
 	 * Test method for {@link uk.gov.hmcts.befta.dse.ccd.DefinitionConverter#main(java.lang.String[])}.
 	 */
 	@Test
 	void testMainIncorrectArgsBool() {
-		String[] args = {"to-json","definition_files","null","tru1e"};
+        String[] args = { "to-json", "definition_files", CcdEnvironment.PREVIEW.name(), "null", "tru1e" };
 		IllegalArgumentException aeThrown = Assertions.assertThrows(IllegalArgumentException.class, () ->
         	DefinitionConverter.main(args),
 				"IllegalArgumentException is not thrown");
-		assertTrue(aeThrown.getMessage().contains("Forth arg should be a boolean but got: "));
+        assertTrue(aeThrown.getMessage().contains("Fifth arg should be a boolean but got: "));
 	}
 
 	/**
 	 * Test method for {@link uk.gov.hmcts.befta.dse.ccd.DefinitionConverter#main(java.lang.String[])}.
 	 */
 	@Test
 	void testMainArgsTojsonTrue() {
-		String[] args = {"to-json",DEFAULT_DEFINITIONS_PATH_EXCEL,TEMPORARY_DEFINITION_FOLDER_JSON,"true"};
+        String[] args = { "to-json", DEFAULT_DEFINITIONS_PATH_EXCEL, CcdEnvironment.PREVIEW.name(),
```

Comment:
If `environment argument is not require/used for "to-json" calls then it should not be in tests.